CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+CHANGES IN VERSION 1.14.10 (FROM 1.13.0)
+===================================
+
+ Appium 1.14.0 is a minor release.
+
+ #### General
+* Minimum Node version is bumped up to v10
+* The default driver for Android is now set to [UiAutomator2](https://github.com/appium/appium-uiautomator2-driver). If the [UiAutomator1](https://github.com/appium/appium-android-driver) driver is desired, then this can be achived by setting `automationName=UiAutomator1` in the capabilities
```

Comment:
@umut I think this should be moved to the top of general and have `IMPORTANT:` prefixing it. This could break users tests if they don't know the proper migration path.

I would also mention that it's being bumped up from UiAutomator1 to UiAutomator2

Also, typo in `achived`.

Reply:
Will do. Thanks for noticing the typo